nilearn/decoding/tests/test_surf_searchlight.py

```diff
@@ -13,7 +14,7 @@
 
 
 from nilearn.surface.tests.test_surface import _generate_surf
-from nilearn.decoding.surf_searchlight import SurfSearchLight
+from nilearn.decoding.surf_searchlight import SurfSearchLight, _apply_surfmask_and_get_affinity
 
 def _create_toy_mesh():
```

Comment:
Add a docstring here. That helps other developers quickly understand the code base.